docs/api/index.md

```diff
@@ -2917,7 +3008,45 @@ IntersectionObserver === undefined
 
 - **Type**: `(path: string) => void`
 
-  Removes module from mocked registry. All subsequent calls to import will return original module even if it was mocked.
+  Removes module from mocked registry. All calls to import will return original module even if it was mocked before. This call is hoisted (moved) to top of the file, so it will only unmock modules that were defined in `setupFiles`, for exampl.
+
+### vi.doUnmock
+
+- **Type**: `(path: string) => void`
+
+  The same as [`vi.unmock`](#vi-unmock), but is not hoisted to top of the file. The next import of the module will import original module instead of the mock. This will not unmock previously imported modules.
```

Comment:
```suggestion
  The same as [`vi.unmock`](#vi-unmock), but is not hoisted to the top of the file. The next import of the module will import the original module instead of the mock. This will not unmock previously imported modules.
```